### ISSUE TITLE
Typescript type declaration indicates that bigint is not supported

### ISSUE THREAD

**Issue by janeklb** [royNiladri/js-big-decimal#77]
**Describe the bug**

Cannot use `bigint` values with big decimal

**To Reproduce**

```ts
import bigDecimal from 'js-big-decimal';

new bigDecimal(10n); // type error
```

**Expected behavior**

no type error :) 

**Additional context**

This seems to be just a TS / typing issue -- `bigint` values look like they work just fine if using plain JS.

**Comment by royNiladri** [comment]
Fixed in v1.3.7. Please verify

**Comment by janeklb** [comment]
Verified - thanks!
